test/grid_animation.jl

```diff
@@ -0,0 +1,158 @@
+            start_point = Point(min_width, y_point)
+            end_point = start_point + step * (Point(max_width, y_point) - start_point)
+            line(start_point, end_point, :stroke)
+        end
+
+        for x_point = min_height:25:max_height
+            step = (frame - first(action.frames)) / (length(action.frames) - 1)
+            start_point = Point(x_point, min_height)
+            end_point = start_point + step * (Point(x_point, max_height) - start_point)
+            line(start_point, end_point, :stroke)
+        end
+
+    elseif grid_direction == :BL
+
+        for y_point = min_width:25:max_width
+            step = (frame - first(action.frames)) / (length(action.frames) - 1)
+            start_point = Point(max_width, y_point)
+            end_point = start_point + step * (Point(min_width, y_point) - start_point)
+            line(start_point, end_point, :stroke)
+        end
+
+        for x_point = min_height:25:max_height
+            step = (frame - first(action.frames)) / (length(action.frames) - 1)
+            start_point = Point(x_point, min_height)
+            end_point = start_point + step * (Point(x_point, max_height) - start_point)
+            line(start_point, end_point, :stroke)
+        end
+
+    elseif grid_direction == :TR
+
+        for y_point = min_width:25:max_width
+            step = (frame - first(action.frames)) / (length(action.frames) - 1)
+            start_point = Point(min_width, y_point)
+            end_point = start_point + step * (Point(max_width, y_point) - start_point)
+            line(start_point, end_point, :stroke)
+        end
+
+        for x_point = min_height:25:max_height
+            step = (frame - first(action.frames)) / (length(action.frames) - 1)
+            start_point = Point(x_point, max_height)
+            end_point = start_point + step * (Point(x_point, min_height) - start_point)
+            line(start_point, end_point, :stroke)
+        end
+
+    elseif grid_direction == :TL
+
+        for y_point = min_width:25:max_width
+            step = (frame - first(action.frames)) / (length(action.frames) - 1)
+            start_point = Point(max_width, y_point)
+            end_point = start_point + step * (Point(min_width, y_point) - start_point)
+            line(start_point, end_point, :stroke)
+        end
+
+        for x_point = min_height:25:max_height
+            step = (frame - first(action.frames)) / (length(action.frames) - 1)
+            start_point = Point(x_point, max_height)
+            end_point = start_point + step * (Point(x_point, min_height) - start_point)
+            line(start_point, end_point, :stroke)
+        end
+
+
+    end
+
+end
+
+function zero_lines(video::Video, action::Action, frame::Int; line_direction::Symbol = :TR)
+
+    min_width = video.width / -2
+    max_width = video.width / 2
+
+    min_height = video.height / -2
+    max_height = video.height / 2
+
+    setline(10)
```

Comment:
Should be variable for the user later but I guess you know that.

Reply:
Yup!